tests/test_utils.h

```diff
@@ -28,3 +28,5 @@ void expectEqualErrors(const std::vector<std::string> &errors, const libcellml::
 
 libcellml::ModelPtr createModel(const std::string &name = "");
 libcellml::ModelPtr createModelWithComponent(const std::string &name = "");
+
+#define EXPECT_EQ_ERRORS(errors, logger) {SCOPED_TRACE("Error occured here");expectEqualErrors(errors, logger);}
```

Comment:
Nice one, I like it! :) Might just want to add a full stop in the trace message?